doc/events.lua

```diff
@@ -0,0 +1,12 @@
+-- You can call me with something like `ldoc --filter events.filter .`
+return {
+   filter = function (t)
+      for _, mod in ipairs(t) do
+         for _, item in ipairs(mod.items) do
+            if item.type == 'event' then
+               print(mod.name,item.name,mod.file,item.lineno)
```

Comment:
@zwim The output from this is:

```
apps.reader.modules.readertoc	TocReset	/home/frans/src/kobo/koreader/frontend/apps/reader/modules/readertoc.lua	115
```

You might need something like that for a giant overview.